extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -494,6 +499,11 @@ export const RoundTripSqlDbPresentCondition = '\'$(NetCoreBuild)\' != \'true\' A
 export const RoundTripSqlDbNotPresentCondition = '\'$(NetCoreBuild)\' != \'true\' AND \'$(SQLDBExtensionsRefPath)\' == \'\'';
 export const DacpacRootPath = '$(DacPacRootPath)';
 export const ProjJsonToClean = '$(BaseIntermediateOutputPath)\\project.assets.json';
+export const EmptyConfigurationCondition = '\'$(Configuration)\' == \'\'';
+export const EmptyPlatformCondition = '\'$(Platform)\' == \'\'';
+export function ConfigurationPlatformCondition(configuration: string, platform: string) { return `'$(Configuration)|$(Platform)' == '${configuration}|${platform}'`; }
```

Comment:
When checking the PropertyGroup condition when looking for OutputPath, only the conditions that are in legacy .sqlproj by default when creating a project in VS and ADS are handled for now.

Reply:
Just checking if you created an issue for the remaining part of the issue?

Reply:
just opened an issue for it now #20439 :)